crates/sui-types/src/messages_grpc.rs

```diff
@@ -395,6 +395,31 @@ impl SubmitTxResponse {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, Hash, Eq, PartialEq)]
+pub struct ConsensusTransactionPosition {
+    pub transaction_digest: TransactionDigest,
+    pub leader_round: u64,
+    pub block_authority_index: u32,
+    pub transaction_index: u32,
+}
+
+// TODO: Use protobuf for the request/response types.
```

Comment:
Would you like some help setting up using protobuf here over bcs?

Reply:
Don't worry we have Xun covered 😀